scripts/cpplint.py

```diff
@@ -1145,6 +1145,7 @@ def RepositoryName(self):
             os.path.exists(os.path.join(current_dir, ".hg")) or
             os.path.exists(os.path.join(current_dir, ".svn"))):
           root_dir = current_dir
+          break;
```

Comment:
Because we want to find the git repository immediately containing the working directory, not the one closest to root. Otherwise this doesn't match up with the folder used by the git diff.